src/collectd-snmp.pod

```diff
@@ -408,6 +439,70 @@ Configures the size of SNMP bulk transfers. The default is 0, which disables bul
 
 =back
 
+=head1 SECURITY
+
+SNMP provides various security levels, ranging from open SNMPv1 and SNMPv2c,
+to the secure SNMPv3 USM and TSM options.
+
+=head2 SNMPv1 / SNMPv2c Security
+
+When B<Version> 1 or 2 is used, anyone with knowledge of the community
+string can connect to the SNMP server.
+
+No authentication or privacy is supported in these modes.
+
+=head2 SNMPv3 USM Security
```

Comment:
You've used USM all over without writing at least once what it means.  While this is not rfc3414, it would be good to give a little more background here, what do you think?